client/my-sites/checkout/src/components/wp-checkout-order-summary.tsx

```diff
@@ -56,10 +56,9 @@ import type { TranslateResult } from 'i18n-calypso';
 // This will make converting to TS less noisy. The order of components can be reorganized later
 /* eslint-disable @typescript-eslint/no-use-before-define */
 
-export default function WPCheckoutOrderSummary( {
+export function WPCheckoutOrderSummary( {
```

Comment:
While it's a good idea to not use default exports, this is also used in some tests. That's why the tests are failing, I think.

Reply:
I updated the import to a named import which seems to have fixed the tests locally, just waiting on Githubs tests to clear.